javatests/arcs/core/data/RecipeTest.kt

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.core.data
+
+import arcs.core.data.Recipe.Handle.Fate
+import arcs.core.storage.StorageKeyParser
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [Recipe]. */
+@RunWith(JUnit4::class)
+internal class RecipeTest {
```

Comment:
Why internal?

Reply:
ah, from copy paste of another test, removed.